kabuki.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python
+# Main kabuki.py
+# imports - use the rpi-rgb-led-matrix instructions to install dependencies
+import queue
+import time
+from multiprocessing import JoinableQueue
+from multiprocessing import Process
+
+from PIL import Image
+from PIL import ImageDraw
+from rgbmatrix import graphics
+
+from face import Face
+
+# constants used throughout project
+WIDTH = 32
+HEIGHT = 32
+SPEED = 1.0 / 20.0
+IDLE_TIME = 50
+EYES = 0
+MOUTH = 16
+
+
+class Kabuki:
+    def __init__(self, matrix):
+        self.matrix = matrix
+        self.face = Face()
+        self.eye_queue = JoinableQueue()
+        self.mouth_queue = JoinableQueue()
+        self.render_queue = JoinableQueue()
+        self.loop_proc = Process(target=self.loop)

```

Comment:
The reason these aren't looping is that the functions you are calling don't loop themselves.

Right now they are processes that have a start and an end, but a process just goes through it linearly without looping.
If you want it to loop, add a while(true) in self.loop, self.compute_frame, self.render_loop.